scripts/eurostat/health_determinants/social_environment/preprocess.py

```diff
@@ -0,0 +1,501 @@
+                                                       pv2=sv_name,
+                                                       pv3=denominator,
+                                                       pv4=education,
+                                                       pv5=residence,
+                                                       pv6=assist,
+                                                       pv7=lev_perc,
+                                                       pv8=gender,
+                                                       pv9=countryofbirth,
+                                                       pv10=citizenship,
+                                                       pv11=lev_limit,
+                                                       pv12=benificiary,
+                                                       pv13=frequency) + "\n"
+
+        # Writing Genereated MCF to local path.
+        with open(self._mcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(final_mcf_template.rstrip('\n'))
+        # pylint: enable=R0914
+        # pylint: enable=R0912
+        # pylint: enable=R0915
+
+    def _generate_tmcf(self) -> None:
+        """
+        This method generates TMCF file w.r.t
+        dataframe headers and defined TMCF template.
+        Args:
+            None
+        Returns:
+            None
+        """
+        # Writing Genereated TMCF to local path.
+        with open(self._tmcf_file_path, 'w+', encoding='utf-8') as f_out:
+            f_out.write(_TMCF_TEMPLATE.rstrip('\n'))
+
+    def process(self):
+        """
+      This Method calls the required methods to generate
+      cleaned CSV, MCF, and TMCF file
+      """
+        final_df = pd.DataFrame(columns=['time','geo','SV','observation',\
+            'Measurement_Method'])
+        # Creating Output Directory
+        output_path = os.path.dirname(self._cleaned_csv_file_path)
+        if not os.path.exists(output_path):
+            os.mkdir(output_path)
+        sv_list = []
+        for file_path in self._input_files:
+            df = pd.read_csv(file_path, sep='\t', header=0)
+            file_name = file_path.split("/")[-1][:-4]
+            function_dict = {
+                "hlth_ehis_ss1e": _hlth_ehis_ss1e,
+                "hlth_ehis_ss1u": _hlth_ehis_ss1u,
+                "hlth_ehis_ic1e": _hlth_ehis_ic1e,
+                "hlth_ehis_ic1u": _hlth_ehis_ic1u,
+                "hlth_ehis_ss1b": _hlth_ehis_ss1b,
+                "hlth_ehis_ss1c": _hlth_ehis_ss1c,
+                "hlth_ehis_ss1d": _hlth_ehis_ss1d
+            }
+            df = function_dict[file_name](df)
+            df['SV'] = df['SV'].str.replace('_Total', '')
+            final_df = pd.concat([final_df, df])
+            sv_list += df["SV"].to_list()
+
+        final_df = final_df.sort_values(by=['time', 'geo', 'SV', 'observation'])
+        final_df = final_df.drop_duplicates(subset=['time','geo','SV'],\
+            keep='first')
```

Comment:
Please remove backslash. Let lint handle formatting. (Please do it in all places where backslash is present.)

Reply:
Removed backslash.